### PR TITLE
initial draft of pingfederate procedure

### PR DESCRIPTION
**Pull request by MBishop17**

### CODE REVIEW

File: content/rancher/v2.x/en/admin-settings/authentication/ping-federate/_index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+>**Prerequisites:**
+>
+>- You must have a [Ping IdP Server](https://www.pingidentity.com/) configured.
+>- Export a `metadata.xml` file from your IdP Server. For more information, see the [PingIdentity documentation](https://documentation.pingidentity.com/pingfederate/pf83/index.shtml#concept_exportingMetadata.html).
+
+1.	From the **Global** view, select **Security > Authentication** from the main menu.
+
+1.	Select **PingIdentity**.
+
+1.	Complete the **Configure Ping Account** form. Rancher can work with your Ping IdP that has Active Directory to manage the users and groups.
```
Comment (mrajashree) [inline content/rancher/v2.x/en/admin-settings/authentication/ping-federate/_index.md]: @MBishop17 Ping can accept databases as well, other than AD, I had forgotten to include it in my previous PR. So can we say this:
Ping Identity Provider lets you specify what data store you want to use. You can either add a database or use an existing ldap server. Let's say for instance you select your Active Directory (AD) server, this is how you can map AD attributes to fields within Rancher

File: content/rancher/v2.x/en/admin-settings/authentication/ping-federate/_index.md [hunk 2]
```diff
@@ -43,4 +43,9 @@ If your organization uses Ping Identity Provider (IdP) for user authentication,
...
     >**Note:** You may have to disable your popup blocker to see the IdP login page.
 
-**Result:** Rancher is configured to work with PingIdentity. Your users can now sign into Rancher using their PingIdentity logins.
\ No newline at end of file
+**Result:** Rancher is configured to work with PingIdentity. Your users can now sign into Rancher using their PingIdentity logins.
+
+>**Ping Identity Provider Caveats:** 
+>
+>- IdP does not support search or lookup. When adding users to [clusters]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/editing-clusters/) or [projects]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/projects-and-namespaces/editing-projects/), the exact IDs must be entered correctly.
+>- When adding users to [clusters]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/editing-clusters/) or [projects]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/projects-and-namespaces/editing-projects/), group IDs are not supported unless the admin who turned on access control is a member of the group.
```
Comment (mrajashree) [inline content/rancher/v2.x/en/admin-settings/authentication/ping-federate/_index.md]: Can we also add this? "When adding a group that admin is a member of, make sure to add it from the dropdown and not from the search bar"

Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/authentication/ping-federate/_index.md]: is this just a pingfederate thing? or is it something you have to do with all auth providers?

File: content/rancher/v2.x/en/k8s-in-rancher/editing-clusters/_index.md [hunk 3]
```diff
@@ -7,10 +7,12 @@ weight: 3015
 
 Following cluster creation, you can add users as cluster members so that they can access its resources.
 
->**Ping Identity Provider Caveats:**
+>**IdP and MS FS Caveats:**
 >
```
Comment (mrajashree) [inline content/rancher/v2.x/en/k8s-in-rancher/editing-clusters/_index.md]: Should this be Ping and ADFS?

**Pull request merged**